CRM/CiviAwards/Helper/CreateReviewFieldsValidator.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/**
+ * Class CRM_CiviAwards_Helper_CreateReviewFieldsValidator.
+ */
+class CRM_CiviAwards_Helper_CreateReviewFieldsValidator {
+
+  /**
+   * Array of accepted review_fields with their validation filters
+   *
+   * @var array
+   */
+  private $reviewFieldsLegalOptions = [
+    'id' => FILTER_VALIDATE_INT,
+    'weight' => FILTER_VALIDATE_INT,
+    'required' => FILTER_VALIDATE_BOOLEAN
+  ];
+
+  /**
+   * Array to hold invalid field options
+   *
+   * @var array
+   */
+  private $invalidReviewFieldOptions = [];
+
+  /**
+   * Array to hold unknown review field options passeed
+   *
+   * @var array
+   */
+  private $unknownReviewFieldOptions = [];
+
+  /**
+   * Validates review_field options to ensure legal options are passed.
+   *
+   * @param array $reviewFields
+   *  Review fields
```

Comment:
Can you PLEASE make sure you run the linters locally. Tried committing this locally and got 15 errors.
This purpose of running it is to make the PR review process smoother and make it easier for the reviewer. If we commit this file this way, the next person working on it will have to deal with fixing the errors you could have fixed.